spark/src/main/scala/org/apache/spark/sql/delta/TableFeature.scala

```diff
@@ -353,6 +353,8 @@ object TableFeature {
         TestFeatureWithDependency,
         TestFeatureWithTransitiveDependency,
         TestWriterFeatureWithTransitiveDependency,
+        // identity columns are under development and only available in testing.
```

Comment:
```suggestion
        // Identity columns are under development and only available in testing.
```

Reply:
Addressed in https://github.com/delta-io/delta/pull/2859 :)